openwisp_monitoring/device/static/monitoring/js/device-map.js

```diff
@@ -246,6 +232,27 @@
                 map.setMaxBounds(
                     L.latLngBounds(L.latLng(-90, -540), L.latLng(90, 540))
                 );
+                map.on('moveend', event => {
+                    let netjsonGraph = this;
+                    let bounds = event.target.getBounds();
+                    if ((bounds._southWest.lng < -180) && !(netjsonGraph.westWorldFeaturesAppended)) {
```

Comment:
```suggestion
                    if (bounds._southWest.lng < -180 && !netjsonGraph.westWorldFeaturesAppended) {
```
Wouldn't this work too?

Reply:
I added extra parenthesis for readability.